### PR TITLE
Open API Project - Update-1

### PR DESCRIPTION
**Pull request by vankampenc**
updated fetch, cleaned up variables, and updated styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,12 +15,43 @@
...
+    <nav>
+
+    </nav>
 </header>
 
 <main>
 
     <h1> Weather  </h1>
 
+<!-- City Selector -->
```
Comment (mannsn) [inline index.html:27]: I was a little confused with your radio buttons..  You have to select the one already selected to make it switch?

**Review by mannsn — APPROVED** [review]
Looking good!  Added comment about radio button.   See two queries.

File: index.html (hunk 1 above)
Comment (vankampenc) [reply index.html:27]: Hi Suzy, I am using chrome and not experiencing this issue. Would you be able to tell me what I could do to reproduce the issue?  Thanks!

Comment (mannsn) [reply index.html:27]: It is now working, not sure why I had the issue before.  My bad!    All looks good!

**Pull request merged**